functions.php

```diff
@@ -184,6 +184,19 @@ function hello_elementor_add_description_meta_tag() {
 }
 add_action( 'wp_head', 'hello_elementor_add_description_meta_tag' );
 
+if ( ! function_exists( 'hello_elementor_settings_panel' ) ) {
+	/**
+	 * Add theme settings panel.
+	 *
+	 * @return void
+	 */
+	function hello_elementor_settings_panel() {
+		include_once get_template_directory() . '/includes/settings/settings-panel.php';
+		new HelloElementor\Includes\Settings\Settings_Panel();
```

Comment:
@rami-elementor Maybe we should thinking about covert it to static methods?

Reply:
It's not a class method, it's a stand alone function triggered by the `init` hook.